logd/store/cursor.go

```diff
@@ -0,0 +1,75 @@
+package store
+
+import (
+	"encoding/base32"
+	"errors"
+	"fmt"
+
+	"github.com/deref/exo/util/bitutil"
+)
+
+var encoding32 = base32.NewEncoding("0123456789abcdefghjkmnpqrstvwxyz").WithPadding(base32.NoPadding)
```

Comment:
I have this same encoding (I believe) in the `gensym` package. Maybe we need a base32util package?